tests/submodule/lookup.c

```diff
@@ -445,3 +445,20 @@ void test_submodule_lookup__foreach_in_bare_repository_fails(void)
 
 	cl_git_fail(git_submodule_foreach(g_repo, foreach_cb, NULL));
 }
+
+void test_submodule_lookup__fail_invalid_gitignores(void)
```

Comment:
`gitignores`? Don't you mean `gitmodules`?

Reply:
Yeah a type, this is still WIP as I can't test locally ATM...